scripts/restore-sanitised-database.sh

```diff
@@ -79,13 +79,19 @@ case $ACCOUNT in
 esac
 
 
+read -p "Do you want to restore the database locally? (y/n): " RESTORE_DB
```

Comment:
I think this question is in the wrong place. If you're restoring locally, then you don't need to do the previous stuff asking the user to select dev or test, so it could be higher up.

Note that when the sanitised DB dump is stored in the prod bucket, the user will still need to select their production aws-vault profile (for both local and AWS restore) - but not the dev/test environment.

Reply:
If we don't select the env at this stage then how would we  load the file from S3, I assumed we need it to download from the exact env but tbh not sure how this will work in prod :)

Reply:
When this script is 'ready', there will be two flows:

1) Pull the sanitised dump from the production account and restore it locally
2) Pull the sanitised dump from the production account and restore it to either `dev` or `test`.

In both cases, we need to ask for the `aws-vault` for `production`. In the second case, we _also_ need to ask for the `aws-vault` profile for the `dev` or `test` account.